composer.json

```diff
@@ -7,6 +7,7 @@
     "enqueue/enqueue": "*@dev",
     "enqueue/stomp": "*@dev",
     "enqueue/amqp-ext": "*@dev",
+    "enqueue/amqplib": "*@dev",
```

Comment:
Could you please rename it to `amqp-lib`? Same goes for the composer package name